tests/HttpClientInterception.Tests/Examples.cs

```diff
@@ -627,5 +628,44 @@ public static async Task Intercept_Http_Get_For_Json_Object_Using_System_Text_Js
             content.RootElement.GetProperty("Id").GetInt32().ShouldBe(1);
             content.RootElement.GetProperty("Link").GetString().ShouldBe("https://www.just-eat.co.uk/privacy-policy");
         }
+
+        [Fact]
+        public static async Task Inject_Latency_For_Http_Get_With_Cancellation()
+        {
+            // Arrange
+            var latency = TimeSpan.FromMilliseconds(50);
+
+            var builder = new HttpRequestInterceptionBuilder()
+                .ForHost("www.google.co.uk")
+                .WithInterceptionCallback(async (_, ct) =>
```

Comment:
Maybe go with `cancellationToken` or `token` to make it clearer as it's an example?